python/eos/plot/plotter.py

```diff
@@ -63,6 +64,34 @@ def next_color(self):
         self.__next_color = (self.__next_color + 1) % len(self.colors)
         return(result)
 
+    @staticmethod
+    def weighted_quantiles(values, quantiles, sample_weight=None):
```

Comment:
Can you make this a private method? ``_weighted_quantiles``? Should we move it to somewhere else? Is it relevant to code beyond ``eos.Plotter``?

Reply:
I keep it in `eos.Plotter` for the time being. We can always move it later.